### DISCUSSION TITLE
Ansible - mongodb ci

### DISCUSSION THREAD

**Discussion by ArpanaQASel** [learndevopsonline/learndevopsonline#1924] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/79815612/ff0e3f36-b94f-4ab4-9da6-adccae50e5af)

Mongo DB Server 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/79815612/09867b74-3106-43ee-b514-4dce08630752)

hi Raghu,

I created this yesterday and it worked but today its showing error. pls suggest

yesterday i

**Comment by chaithanya1812** [comment]
Hello @ArpanaQASel ,
## Paste workstation ip-address. And also read this before posting an error message.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/dcc2a67f-5f3d-4ee3-b47d-7b0786ef84f8)

**Reply by ArpanaQASel** [reply]
Ok Got it!

I created frontend and mongodb yesterday and it worked but today When I tried to run the same code mongodb is giving me the following error in the below task -- Load Catalogue Schema

![image](https://github.com/learndevopsonline/learndevopsonline/assets/79815612/baf041af-f2df-4bf3-bdef-cae4232e2d28)

below is the mongodb-->tasks-->main.yml  code


![image](https://github.com/learndevopsonline/learndevopsonline/assets/79815612/fa682495-aee4-4c9f-ad21-eebed2c8c59e)


Workstation Public IP

![image](https://github.com/learndevopsonline/learndevopsonline/assets/79815612/a3d58b09-698d-41c3-9da5-6f38506cdbf3)


Please help!

Thanks,
Arpana

**Comment by chaithanya1812** [comment]
Hello @ArpanaQASel , while checking you stopped the servers.

**Comment by ArpanaQASel** [comment]
Oh I am sorry i tried for some time then logged off. I will try again from
my end and let you know tomorrow. Pls don't close this issue.

Thank you so much🙏
